docs/community/governance.md

```diff
@@ -0,0 +1,106 @@
+  1. Advisors: People who have been an Executive member in the past and are still on the campus. They have seen a lot of KOSS, and the members should use their experience as a resource.
+  1. Alumni: People who have graduated but were Executive members when on campus.
+
+Permanent membership of KOSS mandates serving one year as an Executive (either as Head or Member).
+
+
+## The Story
+
+Members join KOSS in their first year as Core Team members. They spend their time learning technologies, as well as working on raising the awareness of Open Source. At the end of their second year, they take up the role of Executive Heads and Executive Members after a rigorous [Governance Review Week](/docs/community/governance-review-week). The former Executive Heads and Advisors decide for each. After serving for one year, they pass on the baton to the next batch, and they become Advisors while staying on the campus. They actively take part in the discussions, events, and seminars as per their skills and expertise. They graduate to become KOSS Alumni, the role of which is undecided yet.
+
+## The policy of KOSS membership
+
+We recruit UG 1st years in the team, preferably after they have spent one semester on the campus. We can also hire any 2nd year in their third semester, based on recommendations by the members.
+
+In the spring semester of Core Team members, they go through a Governance week discussion and finally become the new Executives (Heads and Members). Former Executives should carefully offboard members who are not a good fit for the upcoming Executive batch. Every single individual should have a net positive impact on the team. See [Onboarding/Offboarding](/docs/community/onboarding-offboarding) for more detail.
+
+
+## Detailed Role Descriptions
+### 1. Executives
+
+> Members in their late second and early third year
+
+### 1.1 Executive Heads
+
+Executive Heads are de facto managers of KOSS. They have complete authority to make decisions and take up initiatives. They manage KOSS public relations on Emails and Social Media. Executive Heads are responsible for inducting new members as well as pruning anyone’s membership as they unanimously decide. In short, Executive Heads are KOSS.
+
+Executive Heads decide the timeline of KOSS. They call the full-body general meetings. Executive Heads lead all the meetings and events unless they explicitly request anyone else to do so.
+
+**All** of KOSS meetings have to be conducted in the knowledge of an Executive Head. Executive Heads have to take steps to prevent any abuse of power or harassment.
+
+Executive Heads regularly consult with the Advisors. They must do it before introducing any change with high impact.
+
+They approve Pull Requests to https://github.com/kossiitkgp/accounts.
+
+### 1.2 Executive Members
+
+Executive Members do not take the lead but have experience in working with KOSS and the erstwhile Executive Heads. Their primary objectives are:
```

Comment:
Why is _erstwhile_ used here? They have history working with the functioning Executive Heads.

Reply:
Cool.